# This service tries to navigate towards the center of the gas.

---
uint8 success
